api/src/utils/cache.ts

```diff
@@ -0,0 +1,10 @@
+import { Response } from "express"
+
+// cache for 1 week.
+// this cache-control header tells the browser to cache it for a week,
+// as well as tells Vercel's Edge CDN to cache via https://vercel.com/docs/edge-network/caching
+export const cacheStaticResponse = (res: Response) =>
```

Comment:
The name is a bit confusing as I was thinking this function stored the request but instead it's just setting the headers for cache control.
Thinking a bit more about it though maybe we can set the headers on a middleware? I think we want to cache everything outside of the auth right?